cinema/models.py

```diff
@@ -4,7 +4,24 @@
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
+    actors = models.ManyToManyField("Actor", related_name="movies")
+    genres = models.ManyToManyField("Genre", related_name="movies")
     duration = models.IntegerField()
 
     def __str__(self):
         return self.title
+
+
+class Actor(models.Model):
```

Comment:
you can add magic str

Reply:
I agree, it's always a good idea to add it